sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -4177,9 +4177,9 @@ ALTER TABLE symptoms_history ADD COLUMN lossofsmell varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (205, 'Added symptoms loss of taste and loss of smell #1936');
 
--- 2020-05-05 Add new symptoms and health conditions #1824
+-- 2020-09-02 changed column name from coughWithHeamoptysis to coughWithHemoptysis #2758
```

Comment:
Revert these changes because they will break existing servers. You need to write a migration script instead that renames the column. NEVER change existing SQL scripts.